web/templates/message_linked_media_tooltip.hbs

```diff
@@ -1,4 +1,4 @@
-<div id="message_inline_image_tooltip">
+<div id="message-linked-media-tooltip">
```

Comment:
Should we remove this ID? It doesn't look to be used for anything.

Reply:
Yes. And we have other tooltips, such as `drafts_tooltip.hbs`, which open only with an unadorned `<div>`, so there's precedent for that.